app/src/main/java/com/stevesoltys/seedvault/settings/SettingsFragment.kt

```diff
@@ -171,6 +172,8 @@ class SettingsFragment : PreferenceFragmentCompat() {
         try {
             backup.isChecked = backupManager.isBackupEnabled
             backup.isEnabled = true
+            // enable call log backups for existing installs (added end of 2020)
```

Comment:
maybe just add today's date here to be clearer, or leave it out since we have git history anyway.

Reply:
I added a vague date here intentionally, so we can remove this eventually, when we can assume that all existing installs have been updated. So I didn't use today's date, because the code won't ship today, but probably much later to users. git history can be a pain if other commits (like reformatting) pollute the history and git blame, so a vague comment seemed best.

Reply:
What do you mean here by "existing install have been updated."? Like to remove this from CallLogProvider in AOSP?

I think we could still just keep it in, it's quite small.

Also I wanted an exact date to avoid confusion given that Android 11 is near - and that this is for Android 10.

Reply:
No I mean that this code is only there to set this flag on existing installs and once all existing installs ran this code, we can remove it. The data is only there to be able to judge later when is a good time for removal. Removal isn't urgent, but the less code the better.

Reply:
Yeah, how we do determine that though?

Agreed, let's revisit this discussion when we think about removing this, for now it's fine.